apps/mockingbird/README.md

```diff
@@ -0,0 +1,56 @@
+
+![](./docs/images/mockingbird_diagram.png)
+
+
+
+#### Prerequisites
+
+To create an internal device, you must first <u>import the corresponding XIF to the SmartServer using the Device Type widget, the CMS REST API or the DTP loader</u>.  Furthermore, when using IzoT Net Server with the SmartServer, the binary version of the resources needs to be added to your Lonworks/types folder.  It is also <u>required that the SmartServer be put into IMM mode</u> prior to instantiating internal devices using the script. In order to achieve external communication using the script, <u>MQTT ports (1883 and 8883) should be opened via the Features Configuration page</u> on the SmartServer web interface. Finally, the <u>SmartServer needs to be a Quad Core</u> (Serial number contains F or higher) and be running 3.64 or later.
+
+#### Usage
+
+MBird.py script can be used to create MBirds on the Mockingbird host. You can use it to instantiate a number of devices of a specific type and with proprietary handle names. The script can be run locally on the MBH or externally on a PC with access to the MBH network. 
+
+After successful MBird creation, the script will generate .csv and .scr files that can be used for provisioning the devices through the CMS or in the NodeUtil tool. Note that the script will create a .csv and .scr files if they don't exist and append the lines to the existing files. To provision newly created MBirds on the UUT, just import the .csv in the CMS, select the devices that appear after importing, and click 'Provision Selected' in the Devices widget. 
+
+The script's help guide can be accessed by running the  `python mbird.py -h` or `python mbird.py --help`
+
+To create MBirds of a specific type and handle use the following command:  
+
+```shell
+python mbird.py -t <Target SmartServer IP> --hdl <Devices handle> -n <Number of devices> -x <Device type XIF id> -d <Device type> -i <LON interface>
+```
+
+- <Target SmartServer IP>  - IP of the Mockingbird host.  If not supplied, default is localhost.
+- <Devices handle> - Base name of the MBirds to be created. Every unique MBird will have the base name and the device number in its name "<name>-<nr>". If the provided device handle is separated with the "-" followed by a number (e.g. , `m-tier2-50`), the MBird creation will start from the provided number and increment `n` times, where `n` is provided in the next argument. If there is more than one dash in the string, then the incrementing occurs after the last dash. For the example with `-n 10` the device handles would be: `m-tier2-50`, `m-tier2-51`, ..., `m-tier2-60`. 
```

Comment:
I would prefer if 'n' meant you want a total of 'n' devices, not n+1.  This would seem to be more customary.  Or perhaps you are taking a precedent from something else?

Reply:
The script actually creates 'n' devices, the device 'm-tier2-60' shouldn't be there. I’m removing it from the docs and putting 'm-tier2-59'.